src/index.js

```diff
@@ -2,17 +2,46 @@ import './style.css';
 
 const ul = document.querySelector('ul');
 const form = document.querySelector('#add-form');
+const refreshBtn = document.querySelector('#refresh');
 
-const addScore = (e) => {
+async function addScore(e) {
   e.preventDefault();
   const input = document.querySelectorAll('input');
-  const li = document.createElement('li');
-  const p = document.createElement('p');
-  p.innerHTML = `${input[0].value}: ${input[1].value}`;
-  li.appendChild(p);
-  ul.appendChild(li);
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/clgFLMtHrYVTrf5UzXrW/scores',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: input[0].value,
+        score: Number(input[1].value),
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    },
+  );
+
   input[0].value = '';
   input[1].value = '';
-};
+}
+
+async function refreshScoreTable() {
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/clgFLMtHrYVTrf5UzXrW/scores',
+  )
+    .then((response) => response.json())
+    .then((json) => {
+      ul.innerHTML = '';
+      json.result.forEach((element) => {
+        const li = document.createElement('li');
+        const p = document.createElement('p');
+        p.innerHTML = `${element.user}: ${element.score}`;
+        li.appendChild(p);
+        ul.appendChild(li);
+      });
```

Comment:
- [ ] Kindly change this and other regular functions to es6 arrow functions. If you open [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) it takes you to point 4 in the project requirements on Javascript best practices. There it encourages the use of ES6. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a minute and also go through [this article on why ES6 arrow functions are so awesome](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html).

Reply:
I forgot it sorry. I already made the change.